src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -867,32 +869,36 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
             // non-coherency updates is 1 then combine coherency updates with those.
             // Otherwise, put all coherency updates in a separate commit.
             bool combineCoherencyWithNonCoherency = (nonCoherencyUpdates.Count == 1);
+            PullRequestDescriptionBuilder PullRequestDescriptionBuilder = new PullRequestDescriptionBuilder(LoggerFactory, description);
```

Comment:
Nitpick: variable should be lowercase
```suggestion
            PullRequestDescriptionBuilder pullRequestDescriptionBuilder = new PullRequestDescriptionBuilder(LoggerFactory, description);
```

Reply:
Also we should do `var` or 
```
PullRequestDescriptionBuilder pullRequestDescriptionBuilder = new(LoggerFactory, description);
```
?

Reply:
I checked and it's all mixed, we use var in some places and not in others

Reply:
I think we use var in "obvious" places like this one where we repeat the type for no reason `T x = new T();`
and keep it in places where it's not obvious `T x = some.method();`

However, we seem to not prefer var based on `.editorconfig` - https://github.com/dotnet/arcade/blob/fc232204747e6249e73b11e27aeec4de58ab00c1/.editorconfig#L53